dry-cli.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   end
 
   spec.add_dependency 'concurrent-ruby', '~> 1.0'
-  spec.add_dependency 'hanami-utils',    '~> 1.3'
+  spec.add_dependency 'dry-inflector',   '~> 0.2.0'
```

Comment:
I'd recommend `~> 0.2` otherwise it won't allow you to install dry-inflector `> 0.2`